cli/src/commands/util.rs

```diff
@@ -62,7 +62,17 @@ pub(crate) struct UtilCompletionArgs {
 
 /// Run backend-dependent garbage collection.
 #[derive(clap::Args, Clone, Debug)]
-pub(crate) struct UtilGcArgs {}
+pub(crate) struct UtilGcArgs {
+    /// Time threshold
+    ///
+    /// By default, only objects and operations older than 2 weeks are pruned.
+    ///
+    /// Only the string "now" can be passed to this parameter. Support for
+    /// arbitrary absolute and relative timestamps will come in a subsequent
+    /// release.
+    #[arg(long, verbatim_doc_comment)]
```

Comment:
perhaps, `verbatim_doc_comment` is copy-paste error

Reply:
Done.